copylot/hardware/cameras/basler/camera.py

```diff
@@ -0,0 +1,179 @@
+from pypylon import pylon as py
+from pypylon import genicam
+import sys
+exitcode = 0
```

Comment:
do we need this global variable?

Reply:
I was just thinking in case the software crashed and it will close and quit ... From my experience, if things are not closed properly, often restart the computer is required and it is quite annoying. The global variable sort of add a protection in __del__() if there is a better way. please let me know

Reply:
if your program crash, there is no guarentee python garbage collector will call `__del__` methods :) so it is a false sense of protection in this case. I agree with your concern and we can think what else we can do to address this unclosed devices concerns.